stubs/tifftools/__init__.pyi

```diff
@@ -0,0 +1,18 @@
+from .constants import Datatype, Tag, TiffDatatype, TiffTag
+from .exceptions import MustBeBigTiffError, TifftoolsError, UnknownTagError
+from .tifftools import read_tiff, write_tiff
+
+__version__: str
+
+__all__ = [
```

Comment:
This is certainly not complete, but it at least covers all the parts of the API that we're currently using.